src/pattern-library/components/patterns/DialogComponents.js

```diff
@@ -160,7 +160,7 @@ export default function DialogComponents() {
 
   const openLongModal = () => {
     const children = (
-      <div style={{ overflow: 'auto' }}>
+      <div className="hyp-scrollbox">
```

Comment:
This demonstrates that this pattern can be used within a Modal to constrain to the modal dimensions (without setting explicit dimensions).